cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaTopicConfigProvider.java

```diff
@@ -58,10 +62,33 @@ public Properties topicConfigs(String topic) {
     }
   }
 
+  @Override
+  public Map<String, Properties> topicConfigs(Set<String> topics) {
+    KafkaZkClient kafkaZkClient = KafkaCruiseControlUtils.createKafkaZkClient(_connectString,
+            ZK_KAFKA_TOPIC_CONFIG_PROVIDER_METRIC_GROUP,
+            ZK_KAFKA_TOPIC_CONFIG_PROVIDER_METRIC_TYPE,
+            _zkSecurityEnabled);
+
+    AdminZkClient adminZkClient = new AdminZkClient(kafkaZkClient);
```

Comment:
Nit: Just for consistency -- can we keep (1) creation of AdminZkClient and (2) closing the AdminZkClient in a `try-finally` block?